.env.example

```
@@ -11,4 +11,15 @@ KAFKA_BOOTSTRAP_SERVERS=
 # Redis
 REDIS_HOST=
-REDIS_PORT=
\ No newline at end of file
+REDIS_PORT=
+
+# Config Server
+CONFIG_USERNAME=
+CONFIG_PASSWORD=
+CONFIG_URL=
+
+DB_USER=(본인 로컬 DB 아이디)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)